src/model.jl

```diff
@@ -74,8 +74,8 @@ function create_model(
         sum(
             representative_periods[rp].weight *
             graph[u, v].variable_cost *
-            flow[(u, v), rp, B_flow] for (u, v) ∈ F, rp ∈ RP,
-            B_flow ∈ graph[u, v].partitions[rp]
+            flow[(u, v), rp, B_flow] *
+            length(B_flow) for (u, v) ∈ F, rp ∈ RP, B_flow ∈ graph[u, v].partitions[rp]
```

Comment:
You need the resolution too, right?

Reply:
Yes, I agree. Thank you for pointing that out